.jshintrc

```diff
@@ -29,5 +29,8 @@
     "maxparams": 3,
 
     // Limit the nesting to three levels for callbacks.
-    "maxdepth": 3
+    "maxdepth": 3,
+
+    // Identify possibly unsafe line breaks.
+    "laxbreak": true
```

Comment:
I found a official issue about this parameter: https://github.com/jshint/jshint/issues/1867, didn't found the alternative JSCS solution yet.